src/test/java/com/jfrog/ide/common/yarn/YarnTreeBuilderTest.java

```diff
@@ -75,11 +67,27 @@ private Object[][] yarnTreeBuilderProvider() {
         };
     }
 
+    @DataProvider
+    private Object[][] findDependencyImpactPathsProvider() {
```

Comment:
Let's move the Provider above its consumer. Moreover, we don't need to use a provider when we have only one test case.